### PR TITLE
add: exploration team page

### LINKED ISSUE #104: Add a Page to Showcase the Explorations Team's Projects
This page can serve as a guide, providing a table format that succinctly outlines each project, accompanied by brief explanations to give users a quick overview. It can also direct users to various pages on the documentation for more in-depth information. Alternatively, we have the option to integrate the Explorations Team repo as a submodule, allowing us to host the README directly on the site.

### PR DESCRIPTION
**Pull request by swarna1101**
resolves #104

### CODE REVIEW

File: docs/exploration.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+---
+
+## Exploring Uncharted Horizons in Modular Blockchains
+
+  <p>Welcome to the home of <b><i>Avail Uncharted</i></b>. This is the heart of exploration within the Avail ecosystem. Our mission is to encourage exciting initiatives and collaborate closely with the community. Through collaboration, we aim to gain deeper insights into the needs of builders within our ecosystem and push beyond the horizons of what is possible.</p>
+
+## Active Projects
+| Project              | Description | Repository |
```
Comment (DannyS03) [inline docs/exploration.md:16]: The "Project" column contains repository names. We should improve this. 

We can either rename "Project" to "Repository" and remove the existing "Repository" column, or keep "Project" but rename each entry into a more formal project name, and then use the entries that currently exist under "Project" to replace the usage of "link" in each row with the actual repository name. I think I'd be in favor of the latter.

Example:

```
| Adapter for Sovereign SDK |  A data availability adapter for sovereign rollups integrated with the Sovereign SDK   | [sovereign-da-adapter](https://github.com/availproject/sovereign-sdk/tree/main) |

```

File: docs/exploration.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+| sovereign-da-adapter  |  Data availability adapter for Sovereign Rollups integrated with sovereign-sdk   | [link](https://github.com/availproject/sovereign-sdk/tree/main) |
+| madara-da-interface | A unified DA interface for Madara Starknet Sequencer to publish data onto a preferred DA Layer such as Avail for a reliable and secure DA    | [link](https://github.com/keep-starknet-strange/madara/pull/1021) | 
+| validium-node and validium-contracts |A production-ready validium based on the Polygon zkEVM stack with Avail as the data availability layer for scalable and secure DA | [link (node)](https://github.com/QEDK/validium-node) <br/>&<br/> [link (contracts)](https://github.com/QEDK/validium-contracts) | 
+| avail-op-stack-adapter    | Avail DA's integration with Optimism's rollup SDK op-stack | [link](https://github.com/availproject/avail-op-stack-adapter) | 
+| rollkit-da-adapter  | A Data availability adapter for Rollkit's modular rollup framework | [link](https://github.com/rollkit/rollkit/pull/1168) | 
+
```
Comment (DannyS03) [inline docs/exploration.md:24]: We should use this page to offer more of a breakdown of each "exploration".

So, we should address:
- What we mean by adapter (DA adapter)
- What we mean by interface (DA interface)
- In a statement or two, what each project is composed of

File: docusaurus.config.js [hunk 3]
```diff
@@ -275,6 +275,14 @@ module.exports = {
               target: '_self',
               rel: null,
             },
+            {
+              href: '/category/exploration-team/',
```
Comment (DannyS03) [inline docusaurus.config.js:279]: I don't think this is part of a category in its current form

File: docusaurus.config.js [hunk 4]
```diff
@@ -275,6 +275,14 @@ module.exports = {
...
+              label: 'Exploration Team',
```
Comment (DannyS03) [inline docusaurus.config.js:280]:
```suggestion
              label: 'Data Availability Explorations',
```

File: docusaurus.config.js [hunk 5]
```diff
@@ -5,7 +5,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 module.exports = {
   title: "Avail Docs",
   tagline: "The official documentation hub for the Avail Project.",
-  url: "https://docs.availproject.org/",
+  url: "https://availproject.github.io/",
```
Comment (DannyS03) [inline docusaurus.config.js:8]: we should not be changing this back

File: docs/exploration.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+id: exploration
```
Comment (DannyS03) [inline docs/exploration.md:2]:
```suggestion
id: explorations
```

File: sidebars.js [hunk 7]
```diff
@@ -143,6 +206,13 @@ module.exports = {
         "api/use-case-validiums",
       ],
     },
+    {
+      type:"doc",
```
Comment (DannyS03) [inline sidebars.js:210]: we just need to reference the doc's id like how the other id's are included

File: sidebars.js [hunk 8]
```diff
@@ -1,7 +1,69 @@
...
+   *
+   * This section includes explainer guides and content on the PIP
+   * framework and PoS governance.
+   *
+   * **********************************************************************
+   *
+   */
 
   avail: [
-    "index",
```
Comment (DannyS03) [inline sidebars.js:4]: how come this is being removed?

File: sidebars.js [hunk 9]
```diff
@@ -1,7 +1,69 @@
+/**
```
Comment (DannyS03) [inline sidebars.js:1]: All of this was previously deleted because it is unused

**Review by DannyS03 — CHANGES_REQUESTED** [review]
Thanks @swarna1101, I added various suggestions.

Another nit is to change the filename from "exploration" to "explorations"

File: docs/exploration.md (hunk 2 above)
Comment (DannyS03) [reply docs/exploration.md:24]: Not really addressed

File: sidebars.js (hunk 9 above)
Comment (DannyS03) [reply sidebars.js:1]: Not addressed

File: sidebars.js (hunk 8 above)
Comment (DannyS03) [reply sidebars.js:4]: Not addressed

File: sidebars.js (hunk 7 above)
Comment (DannyS03) [reply sidebars.js:210]: Not addressed

File: sidebars.js (hunk 8 above)
Comment (swarna1101) [reply sidebars.js]: fixed in the latest commit

File: docs/exploration.md [hunk 10]
```diff
@@ -14,10 +14,27 @@ This page gives an overview of ***Avail Uncharted***. It shows the exploration w
...
+## OpEVM
+OpEVM is an EVM-compatible sovereign optimistic rollup SDK with working fraud proofs and has support for a decentralized sequencer set. OpEVM is designed for efficient and secure transaction processing. It provides a decentralized infrastructure for running a layer-2 (L2) blockchain. OpEVM enables high-throughput, low-latency transaction processing.
+
+## Adapter for Soverign SDK
+Sovereign SDK is a free open source tool for building rollup. Adapters contain the logic integrating 3rd party codebases into the Sovereign SDK. Data availability layer(DA) determines the set of data that are fed into the business logic of rollup. Sovereign SDK have adaptaers for all most all DA layer.
+
+## Madara DA Interface
+The Madara DA Interface built by the explorations team makes it simple for developers using the Madara Sequencer to utilize Avail’s DA layer. This is particularly useful for developers building chains that leverage Starkware’s ZK technology and developers can still settle on Ethereum via Avail’s Data Attestation Bridge.
+
+## Validium Node and Validium Contracts
```
Comment (DannyS03) [inline docs/exploration.md:33]:
```suggestion
## Polygon zkEVM-based Validium

File: docs/exploration.md [hunk 11]
```diff
@@ -14,10 +14,27 @@ This page gives an overview of ***Avail Uncharted***. It shows the exploration w
...
+| Validium Node and Validium Contracts |A production-ready validium based on the Polygon zkEVM stack with Avail as the data availability layer for scalable and secure DA | [validium-node](https://github.com/QEDK/validium-node) <br/>&<br/> [validium-contracts](https://github.com/QEDK/validium-contracts) | 
+| Avail Stack Adapter    | Avail DA's integration with Optimism's rollup SDK op-stack | [avail-op-stack-adapter ](https://github.com/availproject/avail-op-stack-adapter) | 
+| Rollkit DA Adapter  | A Data availability adapter for Rollkit's modular rollup framework | [rollkit-da-adapter ](https://github.com/rollkit/rollkit/pull/1168) | 
 
+## OpEVM
+OpEVM is an EVM-compatible sovereign optimistic rollup SDK with working fraud proofs and has support for a decentralized sequencer set. OpEVM is designed for efficient and secure transaction processing. It provides a decentralized infrastructure for running a layer-2 (L2) blockchain. OpEVM enables high-throughput, low-latency transaction processing.
+
+## Adapter for Soverign SDK
```
Comment (DannyS03) [inline docs/exploration.md:27]:
```suggestion
## Soverign SDK Adapter 
```

File: docs/exploration.md [hunk 12]
```diff
@@ -14,10 +14,27 @@ This page gives an overview of ***Avail Uncharted***. It shows the exploration w
...
+| Adapter for Soverign SDK  | A data availability adapter for Sovereign Rollups integrated with sovereign-sdk   | [sovereign-da-adapter](https://github.com/availproject/sovereign-sdk/tree/main) |
+| Madara DA Interface | A unified DA interface for Madara Starknet Sequencer to publish data onto a preferred DA Layer such as Avail for a reliable and secure DA    | [madara-da-interface](https://github.com/keep-starknet-strange/madara/pull/1021) | 
+| Validium Node and Validium Contracts |A production-ready validium based on the Polygon zkEVM stack with Avail as the data availability layer for scalable and secure DA | [validium-node](https://github.com/QEDK/validium-node) <br/>&<br/> [validium-contracts](https://github.com/QEDK/validium-contracts) | 
+| Avail Stack Adapter    | Avail DA's integration with Optimism's rollup SDK op-stack | [avail-op-stack-adapter ](https://github.com/availproject/avail-op-stack-adapter) | 
+| Rollkit DA Adapter  | A Data availability adapter for Rollkit's modular rollup framework | [rollkit-da-adapter ](https://github.com/rollkit/rollkit/pull/1168) | 
 
+## OpEVM
```
Comment (DannyS03) [inline docs/exploration.md:24]:
```suggestion
## Optimistic EVM Rollup

File: docs/exploration.md [hunk 13]
```diff
@@ -14,10 +14,27 @@ This page gives an overview of ***Avail Uncharted***. It shows the exploration w
...
+OpEVM is an EVM-compatible sovereign optimistic rollup SDK with working fraud proofs and has support for a decentralized sequencer set. OpEVM is designed for efficient and secure transaction processing. It provides a decentralized infrastructure for running a layer-2 (L2) blockchain. OpEVM enables high-throughput, low-latency transaction processing.
```
Comment (DannyS03) [inline docs/exploration.md:25]: We don't mention anywhere how this is built on top of Avail and briefly how it works with Avail.

**Review by DannyS03 — CHANGES_REQUESTED** [review]
We still don't explain what we mean by adapter or interface. As proposed [here](https://github.com/availproject/availproject.github.io), let's include these terms as part of the glossary in another update and not let it be a blocker for this.

The statements on each project could probably be more compact and all need to be improved for writing style. There should also be some consistency in the form they are presented in. However, the idea is to explain the project (i.e, exploration), and not just the specific stacks being used - the "Madara DA Interface" section is a good example.

**Pull request merged**